src/passport/setup.ts

```diff
@@ -24,6 +24,27 @@ passport.deserializeUser((userId: number, done) => {
     })
 })
 
+passport.use(
+  'authtoken',
+  new AuthTokenStrategy(async (token, done) => {
```

Comment:
You need to specify headerFields in options as we'll be authenticating using the headers.

Use x-Authorization-Token as the key